src/Apps/IdentityVerification/__tests__/IdentityVerificationApp.test.tsx

```diff
@@ -134,13 +173,13 @@ describe("IdentityVerification route", () => {
         const page = await env.buildPage()
         env.mutations.mockNetworkFailureOnce()
 
-        await page.clickStartVerification()
-
-        const errorModal = expectOne(page.find(ErrorModal))
-        expect(errorModal.props().show).toBe(true)
-        expect(page.text()).toContain(
-          "Something went wrong. Please try again or contact verification@artsy.net."
-        )
+        page.clickStartVerification().then(() => {
```

Comment:
For my own education – why the change the `await` to `then` here? Does this have to do with the Jest error that you tracked down? Is it just for consistency purposes?

Reply:
Yes, i think this was while i was trying to debug errors in our tests. As it turns out this didn't fix the issue so i'll fix now.